src/marks/index.js

```diff
@@ -74,10 +74,39 @@ class Strike extends TipTapStrike {
 
 }
 
+class Link extends TipTapLink {
+
+	get schema() {
+		return {
+			attrs: {
+				href: {
+					default: null,
+				},
+			},
+			inclusive: false,
+			parseDOM: [
+				{
+					tag: 'a[href]',
+					getAttrs: dom => ({
+						href: dom.getAttribute('href'),
+					}),
+				},
+			],
+			toDOM: node => ['a', {
+				...node.attrs,
+				title: node.attrs.href,
```

Comment:
This line is the only difference to the tiptap implementation

Reply:
https://github.com/scrumpy/tiptap/blob/master/packages/tiptap-extensions/src/marks/Link.js